website/docs/docs/build/udfs.md

```diff
@@ -49,7 +49,7 @@ To define UDFs in dbt, refer to the following steps:
 
     ```sql
 
-    REGEXP_CONTAINS(a_string, r'^[0-9]+$')
+    REGEXP_INSTR(a_string, r'^[0-9]+$')
```

Comment:
what dialect is this for? 

<img width="720" height="203" alt="Screenshot 2025-11-04 at 3 44 07 PM" src="https://github.com/user-attachments/assets/e3e0fd70-5566-4153-8b20-885228034078" />

this quoting did not work for me in snowflake i had to change this to `REGEXP_INSTR(a_string, '^[0-9]+$')`

cc: @QMalcolm